Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM gradle:8.10-jdk21-jammy AS base
+
+WORKDIR /app/
+
+COPY build.gradle settings.gradle .
+COPY src/ ./src/
+
+EXPOSE 8080
+
+CMD ["gradle", "build"]
+
+FROM gradle:8.10-jdk21-jammy as build
+
+WORKDIR /app/
+
+COPY --from=base /app .
+
+RUN gradle bootJar
+
+FROM bitnami/java:21 AS deployment
```

Comment:
Wasn't entirely sure what Java image to use. OpenJDK images seem to be deprecated. This seems to work fine.